rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/util/PackageNameConverter.groovy

```diff
@@ -123,29 +212,53 @@ class PackageNameConverter {
     }.findAll{it.name}
   }
 
-  public static String buildAppVersionStr(BakeRequest bakeRequest, OsPackageName osPackageName) {
+  public static String buildAppVersionStr(BakeRequest bakeRequest, OsPackageName osPackageName, BakeRequest.PackageType packageType) {
     // As per source of AppVersion, these are valid appversion tags:
     //   subscriberha-1.0.0-h150
     //   subscriberha-1.0.0-h150.586499
     //   subscriberha-1.0.0-h150.586499/WE-WAPP-subscriberha/150
+    //   ContosoUtilities.ja-JP.1.0.0-rc1+asdfe
```

Comment:
I've removed that comment as it shouldn't have been included. I'll check the link and verify. We haven't seen any issues yet but doesn't mean they don't exist.